src/test/java/kb/weather/WeatherServiceApplicationTests.java

```diff
@@ -1,13 +1,13 @@
-package kn.demo;
+package kb.weather;
 
 import org.junit.jupiter.api.Test;
 import org.springframework.boot.test.context.SpringBootTest;
 
 @SpringBootTest
-class DemoApplicationTests {
+class WeatherServiceApplicationTests {
 
     @Test
     void contextLoads() {
+        // todo we need tests
```

Comment:
we need tests

Reply:
As @MisQl said, many tests 😄

Reply:
why you resolved this comment?

Reply:
I've done some tests and I thought I should mark "resolve".